plugins/s3-viewer-backend/package.json

```diff
@@ -45,6 +45,7 @@
     "@backstage/plugin-permission-node": "^0.7.21",
     "@backstage/types": "^1.1.1",
     "@spreadshirt/backstage-plugin-s3-viewer-common": "^0.3.9",
+    "@spreadshirt/backstage-plugin-s3-viewer-node": "0.0.0",
```

Comment:
I thought the same when doing it to the upstream backstage. However, setting it to 0.0.0 seems to be the way to go, because the changeset plugin will apply the `minor` and the released version will be 0.1.0 as expected